tap-schema/build-all

```diff
@@ -4,9 +4,21 @@
 # values that control the presentation order of schemas.  This is respected by the Portal Aspect.
 #
 ./build mock
-./build idfprod ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
-./build idfint ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
-./build idfdev ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
-./build idfsso ../yml/dp03.yaml
-./build usdf-dev ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/oga_live_obscore.yaml
-./build usdf-prod ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml
+
+./build tap-schema-idfprod-tap ../yml/dp02_dc2.yaml ../yml/dp01_dc2.yaml
+./build tap-schema-idfint-tap ../yml/dp02_dc2.yaml ../yml/dp01_dc2.yaml
+./build tap-schema-idfdev-tap ../yml/dp02_dc2.yaml ../yml/dp01_dc2.yaml
+./build tap-schema-usdf-dev-tap ../yml/dp02_dc2.yaml
+./build tap-schema-usdf-prod-tap ../yml/dp02_dc2.yaml
+
+./build tap-schema-idfprod-obstap ../yml/dp02_obscore.yaml
+./build tap-schema-idfint-obstap ../yml/dp02_obscore.yaml
+./build tap-schema-idfdev-obstap ../yml/dp02_obscore.yaml
+./build tap-schema-usdf-dev-obstap ../yml/dp02_obscore.yaml ../yml/oga_live_obscore.yaml
+./build tap-schema-usdf-prod-obstap ../yml/dp02_obscore.yaml
+
+./build tap-schema-idfprod-sso ../yml/dp03.yaml
```

Comment:
The SSOs are fine, but I don't really see a need to have five separate ones, as there is only one back-end DB.

Reply:
So that they can have different schemas as we test things and move it around, it doesn't hurt us and it follows the pattern

Reply:
which one is the backend DB? i thought we had a bunch?

Reply:
> which one is the backend DB? i thought we had a bunch?

As far as I know there is currently only one DP0.3 / SSO database - a Postgres DB at SLAC.

It's not wrong to have five identical TAP_SCHEMA builds for this, and I can think of some ways we might use them, conceivably, so I'm not asking you to remove them, but just noting that it's flexibility we might not need.

Reply:
And how many environments do you want connected to that SSO database?  just usdf-dev? usdf-dev and usdf-prod?  I want to know not only which databases exist but which tap services exist

Reply:
Based on @fritzm's response on #dm-rsp-team this morning, I think the answer is that we want an SSO-only TAP service on all five of idf-prod, idf-int, idf-dev, usdf-prod, and usdf-dev.  All of them will normally point to the same SSO-only Postgres database at USDF, except that during the upcoming USDF outage the IDF RSPs' SSO TAP services will point to the replica of that database recently set up by Dan in the Google cloud.  (See Slack for more discussion of the timing of the switchover.)

As I said above, I don't foresee any reason for the TAP_SCHEMA content of the five to be different from each other, but if you want to create five images as a hedge, I don't object.